tasks/tinc_configure.yml

```diff
@@ -103,3 +125,25 @@
   systemd:
     name: "{{ tinc_service_name }}"
     enabled: yes
+  when: ansible_service_mgr == "systemd"
```

Comment:
we can remove the conditional here, assuming the next task is moved (templating the init for openwrt): Everything can be using service to ensure enabled, and then a restart would be triggered by the handler.